### PR TITLE
CORE-15403: Update Snyk vulnerability

### PR DESCRIPTION
**Pull request by nargas-ritu**
CORE - 15403: Update Snyk vulnerability

### CODE REVIEW

**Review by vkolomeyko — DISMISSED** [review]
LGTM

**Review by yarivus — DISMISSED** [review]
LGTM

File: .snyk [hunk 1]
```diff
@@ -10,14 +10,21 @@ ignore:
           identification algorithm set to “HTTPS”, which means a Corda gateway
           will always validate server certificate hostnames when establishing
           HTTPS connections with other gateways.
-        expires: 2023-10-10T12:08:30.514Z
+        expires: 2023-12-13T12:08:30.514Z
         created: 2022-12-20T12:08:30.517Z
-SNYK-JAVA-ORGECLIPSEJETTY-5769685:
+  SNYK-JAVA-ORGBOUNCYCASTLE-5771340:
```
Comment (yarivus) [inline .snyk]:
```suggestion
SNYK-JAVA-ORGBOUNCYCASTLE-5771340:
```

Let's remain consistent with tabs/spaces

**Review by yarivus — CHANGES_REQUESTED** [review]

**Review by vkolomeyko — APPROVED** [review]
LGTM

**Review by yarivus — APPROVED** [review]
LGTM

**Review by ronanbrowne — APPROVED** [review]
LGTM

**Pull request merged**